raiden/tests/scenarios/mfee4_combined_fees.yaml

```diff
@@ -36,6 +36,7 @@ nodes:
     proportional-imbalance-fee:
       - "0x62083c80353Df771426D209eF578619EE68D5C7A"
       - 20_000  # 2% imbalance fee on channel balance
+    no-cap-mediation-fees: true
```

Comment:
why don't we cap the mediation fees in this scenario? The "normal" case would be with capped fees, right?

Reply:
Since we want to support uncapped mediation fees, I didn't want to remove the scenarios exercising that option. It's also the smaller change. But we can add a copy of this scenario with capping turned on.

Reply:
yes, we need a copy of mfee3 and mfee4 where capping is enabled. Can you create those?

Reply:
I will do so in another PR.

Reply:
I didn't realize that capping the mediation fees does absolutely nothing on the mfee4 scenario. This makes copies of mfee3 and mfee4 with different fee capping settings very boring. Is it still worth it, or should we just remove `no-cap-mediation-fees` from mfee4, because we have to add another fee scenario that exercises the interesting fee capping cases, anyway?